.circleci/base_config.yml

```diff
@@ -323,27 +323,43 @@ jobs:
               MINIMAL_DEBUGINFO="ON"
             fi
 
-            arch="<<parameters.arch>>"
-            if test "$arch" == "x64"; then
-              LAPACK_LIB_PATH="/usr/lib/x86_64-linux-gnu/lapack/liblapack.a"
-              BLAS_LIB_PATH="/usr/lib/x86_64-linux-gnu/blas/libblas.a"
+            # It is not static build if any sanitizer is used
+            if [ -n "<< pipeline.parameters.sanitizer >>" ]; then
```

Comment:
I suppose a dynamic linking is not the subject only to sanitized builds. However, AFAIK there are no other builds of this kind for CircleCI.

Reply:
In practice no, in our CI yes. I am unsure how I would differentiate here whether the static flag was set. Do you have and idea?